squad_planificacion/StationInfo[] stations
